shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz_hints/domain/interactor/StepQuizHintsInteractor.kt

```diff
@@ -30,6 +30,6 @@ class StepQuizHintsInteractor(
             .getUserStorageValue(key = UserStoragePathBuilder.buildSeenHints(stepId), DataSourceType.REMOTE)
             .getOrNull()?.jsonObject?.keys?.map { it.toLong() }
 
-        return commentsDataInteractor.getHintsIDs(stepId).filter { seenHintsIds?.contains(it) ?: true }
+        return commentsDataInteractor.getHintsIDs(stepId).filter { seenHintsIds?.contains(it)?.not() ?: true }
```

Comment:
Давай в несколько строк:
```suggestion
        return commentsDataInteractor
            .getHintsIDs(stepId)
            .filter { seenHintsIds?.contains(it)?.not() ?: true }
```